controllers/crunchybridge/bridgecluster_controller.go

```diff
@@ -39,17 +53,121 @@ type BridgeClusterReconciler struct {
 
 // Reconcile is part of the main kubernetes reconciliation loop which aims to
 // move the current state of the cluster closer to the desired state.
-// TODO(user): Modify the Reconcile function to compare the state specified by
-// the BridgeCluster object against the actual cluster state, and then
-// perform operations to make the cluster state reflect the state specified by
-// the user.
 //
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.8.3/pkg/reconcile
 func (r *BridgeClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	_ = log.FromContext(ctx)
+	logger := log.FromContext(ctx, "BridgeCluster", req.NamespacedName)
+
+	if r.BridgeClient == nil {
```

Comment:
where we are setting API credentials for cluster creation?

Reply:
The reconciler is given a client when initialized in `main` - either a client using configured credentials from the dbaas setup, or a new client configured from being pointed to a secret object identified by environment variables.

Reply:
can we add these few lines in the readme file about configuring the credentails option?

Reply:
Agreed, that makes a lot of sense - good catch